src/folio.md

```
@@ -0,0 +1,3 @@
+# Folio Rebalancer
+
+Folio rebalances a portfolio of tokens to maintain an equal weight distribution across all assets.
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)